cmd/serv.go

```diff
@@ -302,7 +302,7 @@ func runServ(c *cli.Context) error {
 	if _, err := os.Stat(setting.RepoRootPath); err != nil {
 		if os.IsNotExist(err) {
 			return fail("Incorrect configuration.",
-				"Directory `[repository]` `ROOT` was not found, please check if $GITEA_WORK_DIR is passed to the SSH connection or make `[repository]` `ROOT` an absolute value.")
+				"Directory `[repository]` `ROOT` %s was not found, please check if $GITEA_WORK_DIR is passed to the SSH connection or make `[repository]` `ROOT` an absolute value.", setting.RepoRootPath)
```

Comment:
I might be totally wrong here, but wasn't there something that the root directory was intentionally not displayed here so that attackers cannot simply find out where this Gitea instance is stored?
This line seems oddly familiar, I think I once suggested the same thing here, but I'm far from certain.

Reply:
That's correct this was hidden on purpose. Thanks for the reminder @delvh

Reply:
Wow. Interesting that I really remembered this specific suggestion...
And I already wondered why that thought plopped into my head.
I mean, when did I make that suggestion? About half a year ago???

Reply:
Given the number of times people wanted to add the root directory here already, I think it is time for a comment.

Reply:
Sorry, I was wrong at that time that worrying about the path would be leaked to attackers, and might mislead some reviewers. Only the first message of `fail` will be returned to end user, so it's safe to print the path in server's log messages.